lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -732,6 +732,293 @@ class DecomposeAtenTriuOp : public OpRewritePattern<AtenTriuOp> {
 };
 } // namespace
 
+static std::tuple<int64_t, int64_t, int64_t>
+getTrilSizes(PatternRewriter &rewriter, Location loc, int64_t row, int64_t col,
```

Comment:
This function is not needed. It is only used to calculate `getTriuSizes`, and it is more concise to implement `getTriuSizes` directly compared to having both helper functions.

Reply:
I was thinking about that but then I saw that there is another operator torch.tril_indices, which needs to be implemented, uses the function getTrilSizes. So I decided to separate getTrilSizes because it'll be used during lowering of torch.tril_indices.

Reply:
I see, if the lowering for `tril_indices` is future work, then this is fine.